backend/src/main/java/com/google/sps/FindMatchQuery.java

```diff
@@ -16,47 +16,47 @@
 
 import com.google.sps.data.Match;
 import com.google.sps.data.Participant;
+import com.google.sps.datastore.ParticipantDatastore;
 import java.time.Clock;
 import java.time.ZonedDateTime;
 import java.util.List;
+import javax.annotation.Nullable;
 
-/** Class used to find a match in a list of Participants with the most recently added Participant */
+/** Class used to find a match for new Participant with unmatched Participants in datastore */
 public final class FindMatchQuery {
 
   /** Maximum difference in duration to be compatible */
-  private static int MAX_DURATION_DIFF = 15;
-  /** extra padding time to ensure large enough meeting time block */
-  private static int PADDING_TIME = 15;
+  private static final int MAX_DURATION_DIFF = 15;
+  /** Extra padding time to ensure large enough meeting time block */
+  private static final int PADDING_TIME = 10;
```

Comment:
Padding time is used to make sure there's enough time for participants to get ready and join the meet call for the duration before at least one of them is unavailable. I also had to update the tests because I had 2 tests testing edge cases.